runtime/test/test_argument_conversion.cpp

```diff
@@ -289,6 +290,49 @@ void test_recursive(mlir::MLIRContext *ctx) {
   // clang-format on
 }
 
+void test_state(mlir::MLIRContext *ctx) {
+  {
+    std::vector<std::complex<double>> data{M_SQRT1_2, M_SQRT1_2, 0., 0.,
+                                           0.,        0.,        0., 0.};
+    auto x =
+        cudaq::state(new cudaq::FakeSimulationState(data.size(), data.data()));
+    std::vector<void *> v = {static_cast<void *>(&x)};
+    doSimpleTest(ctx, "!cc.ptr<!cc.state>", v);
+  }
+
+  // clang-format off
+// CHECK:       Source module:
+// CHECK:         func.func private @callee(!cc.ptr<!cc.state>)
+// CHECK:       Substitution module:
+
+// CHECK-LABEL:   cc.arg_subst[0] {
```

Comment:
It looks like this test is failing: https://github.com/NVIDIA/cuda-quantum/actions/runs/10499383135/job/29086484483#step:8:742

Reply:
fixed now, thanks!